selfservice/strategy/code/.snapshots/TestVerification-description=should_set_all_the_correct_verification_payloads.json

```diff
@@ -49,5 +36,18 @@
       }
     },
     "type": "input"
+  },
+  {
+    "attributes": {
+      "disabled": false,
+      "name": "csrf_token",
```

Comment:
not sure if re-ordering CSRF token "inputs" will be an issue?

Reply:
It should not matter.